internal/proto/controller/api/services/v1/credential_library_service.proto

```diff
@@ -83,43 +86,43 @@ service CredentialLibraryService {
 }
 
 message GetCredentialLibraryRequest {
-  string id = 1;
+  string id = 1; // @gotags: `class:"public"`
 }
 
 message GetCredentialLibraryResponse {
-  resources.credentiallibraries.v1.CredentialLibrary item = 1;
+  resources.credentiallibraries.v1.CredentialLibrary item = 1; // @gotags: `class:"public"`
 }
 
 message ListCredentialLibrariesRequest {
-  string credential_store_id = 1 [json_name = "scope_id"];
-  string filter = 30 [json_name = "filter"];
+  string credential_store_id = 1 [json_name = "scope_id"]; // @gotags: `class:"public"`
+  string filter = 30 [json_name = "filter"]; // @gotags: `class:"public"`
 }
 
 message ListCredentialLibrariesResponse {
-  repeated resources.credentiallibraries.v1.CredentialLibrary items = 1;
+  repeated resources.credentiallibraries.v1.CredentialLibrary items = 1; // @gotags: `class:"public"`
```

Comment:
Same here (and elsewhere)

Reply:
It looks like repeated fields *do* need to be classified for the classification on the individual items to take effect.

Reply:
nope they dont

Reply:
Should we remove this?

Reply:
yep, missed that one